apps/web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useState } from 'react';
+import './SearchBar.scss';
+
+export interface Props {
+  /** What happens when we use a searchstring to search for results? */
+  handleSearch: (searchString: string) => void;
```

Comment:
Great use of the callback. I think you are starting to really understand separation of business logic from implementation logic.